packages/cli/src/scanner/page.ts

```diff
@@ -100,17 +112,45 @@ export class Page {
             browserSpec: await this.browser.version(),
         };
 
-        if (response.request().redirectChain().length > 0) {
+        if (response.request().redirectChain().length > 0 && response.request().url() !== axeResults.url) {
+            console.log(`Scan performed on redirected page ${axeResults.url}`);
             scanResults.scannedUrl = axeResults.url;
         }
 
         return scanResults;
     }
 
-    public async close(): Promise<void> {
-        if (this.puppeteerPage !== undefined) {
-            await this.puppeteerPage.close();
-            await this.webDriver.close();
+    private async waitForPageToCompleteRendering(page: Puppeteer.Page, timeoutMsecs: number): Promise<void> {
```

Comment:
Will we have this issue in the crawler?

Reply:
yes, this should be added to crawler as well